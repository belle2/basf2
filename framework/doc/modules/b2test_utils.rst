.. automodule:: b2test_utils
   :members:
