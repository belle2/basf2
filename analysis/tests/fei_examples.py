#!/usr/bin/env python3
# -- analysis/tests/fei_examples.py

import b2test_utils as b2tu
import b2test_utils_analysis as b2tua

#: extend the examples test class to actually run over the FEI examples:


class ExamplesTest(b2tua.ExamplesTest):
    def test_fei_examples(self):
        """
        Test supported FEI examples.
        """

        self._test_examples_dir('analysis/examples/FEI/')


if __name__ == '__main__':
    with b2tu.clean_working_directory():
        b2tua.unittest.main()
