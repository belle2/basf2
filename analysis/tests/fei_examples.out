[INFO] Steering file: analysis/tests/fei_examples.py
TFile: B_converted_apply.root
TTree: B0SL
TBranch: __experiment__, 65
TBranch: __run__, 58
TBranch: __event__, 3
TBranch: __production__, 0
TBranch: __candidate__, 0
TBranch: __ncandidates__, 20
TBranch: __eventType__, 
TBranch: __weight__, 1
TBranch: cosThetaBetweenParticleAndNominalB, -1.41
TBranch: isSignalAcceptMissingNeutrino, 0
TBranch: mcErrors, 378
TBranch: extraInfo__bodecayModeID__bc, 4
TBranch: extraInfo__bouniqueSignal__bc, 0
TBranch: extraInfo__boSignalProbability__bc, 0.01637
TTree: B0
TBranch: __experiment__, 65
TBranch: __run__, 58
TBranch: __event__, 5
TBranch: __production__, 0
TBranch: __candidate__, 0
TBranch: __ncandidates__, 20
TBranch: __eventType__, 
TBranch: __weight__, 1
TBranch: Mbc, 5.248
TBranch: deltaE, 0.06644
TBranch: isSignal, 0
TBranch: mcErrors, 444
TBranch: extraInfo__bodecayModeID__bc, 4
TBranch: extraInfo__bouniqueSignal__bc, 0
TBranch: extraInfo__boSignalProbability__bc, 2.052e-06
TTree: BplusSL
TBranch: __experiment__, 65
TBranch: __run__, 58
TBranch: __event__, 3
TBranch: __production__, 0
TBranch: __candidate__, 0
TBranch: __ncandidates__, 20
TBranch: __eventType__, 
TBranch: __weight__, 1
TBranch: cosThetaBetweenParticleAndNominalB, -3.168
TBranch: isSignalAcceptMissingNeutrino, 0
TBranch: mcErrors, 378
TBranch: extraInfo__bodecayModeID__bc, 2
TBranch: extraInfo__bouniqueSignal__bc, 0
TBranch: extraInfo__boSignalProbability__bc, 0.0187
TTree: Bplus
TBranch: __experiment__, 65
TBranch: __run__, 58
TBranch: __event__, 5
TBranch: __production__, 0
TBranch: __candidate__, 0
TBranch: __ncandidates__, 4
TBranch: __eventType__, 
TBranch: __weight__, 1
TBranch: Mbc, 5.211
TBranch: deltaE, -0.4393
TBranch: isSignal, 0
TBranch: mcErrors, 444
TBranch: extraInfo__bodecayModeID__bc, 2
TBranch: extraInfo__bouniqueSignal__bc, 0
TBranch: extraInfo__boSignalProbability__bc, 2.558e-06
TFile: B_generic_apply.root
TTree: B0SL
TBranch: __experiment__, 1003
TBranch: __run__, 0
TBranch: __event__, 1
TBranch: __production__, 0
TBranch: __candidate__, 0
TBranch: __ncandidates__, 20
TBranch: __eventType__, 
TBranch: __weight__, 1
TBranch: cosThetaBetweenParticleAndNominalB, -1.363
TBranch: isSignalAcceptMissingNeutrino, 0
TBranch: mcErrors, 376
TBranch: extraInfo__bodecayModeID__bc, 4
TBranch: extraInfo__bouniqueSignal__bc, 0
TBranch: extraInfo__boSignalProbability__bc, 0.02344
TTree: B0
TBranch: __experiment__, 1003
TBranch: __run__, 0
TBranch: __event__, 1
TBranch: __production__, 0
TBranch: __candidate__, 0
TBranch: __ncandidates__, 4
TBranch: __eventType__, 
TBranch: __weight__, 1
TBranch: Mbc, 5.136
TBranch: deltaE, -0.06938
TBranch: isSignal, nan
TBranch: mcErrors, 512
TBranch: extraInfo__bodecayModeID__bc, 28
TBranch: extraInfo__bouniqueSignal__bc, 0
TBranch: extraInfo__boSignalProbability__bc, 5.43e-07
TTree: BplusSL
TBranch: __experiment__, 1003
TBranch: __run__, 0
TBranch: __event__, 1
TBranch: __production__, 0
TBranch: __candidate__, 0
TBranch: __ncandidates__, 20
TBranch: __eventType__, 
TBranch: __weight__, 1
TBranch: cosThetaBetweenParticleAndNominalB, -0.7587
TBranch: isSignalAcceptMissingNeutrino, 0
TBranch: mcErrors, 376
TBranch: extraInfo__bodecayModeID__bc, 1
TBranch: extraInfo__bouniqueSignal__bc, 0
TBranch: extraInfo__boSignalProbability__bc, 0.0005818
TTree: Bplus
TBranch: __experiment__, 1003
TBranch: __run__, 0
TBranch: __event__, 1
TBranch: __production__, 0
TBranch: __candidate__, 0
TBranch: __ncandidates__, 13
TBranch: __eventType__, 
TBranch: __weight__, 1
TBranch: Mbc, 5.215
TBranch: deltaE, -0.5671
TBranch: isSignal, nan
TBranch: mcErrors, 512
TBranch: extraInfo__bodecayModeID__bc, 31
TBranch: extraInfo__bouniqueSignal__bc, 0
TBranch: extraInfo__boSignalProbability__bc, 4.755e-07
TFile: B_specific_apply.root
TTree: variables
No entry found
TBranch: __experiment__
TBranch: __run__
TBranch: __event__
TBranch: __production__
TBranch: __candidate__
TBranch: __ncandidates__
TBranch: __eventType__
TBranch: __weight__
TBranch: M
TBranch: m2RecoilSignalSide
TBranch: E
